brayns/network/entrypoints/ExportFramesEntrypoint.cpp

```diff
@@ -0,0 +1,223 @@
+/* Copyright (c) 2015-2021 EPFL/Blue Brain Project
+ * All rights reserved. Do not distribute without permission.
+ * Responsible Author: Nadir Roman Guerrero <nadir.romanguerrero@epfl.ch>
+ *
+ * This file is part of Brayns <https://github.com/BlueBrain/Brayns>
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License version 3.0 as published
+ * by the Free Software Foundation.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU Lesser General Public License for more
+ * details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this library; if not, write to the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ */
+
+#include "ExportFramesEntrypoint.h"
+
+#include <brayns/utils/image/ImageEncoder.h>
+#include <brayns/utils/image/ImageFormat.h>
+
+#include <spdlog/fmt/fmt.h>
+
+namespace brayns
+{
+namespace
+{
+void logExportInfo(const ExportFramesParams& info)
```

Comment:
We can specialize fmt::formatter or the << operator of std::ostream with spdlog extension to do that (no need to change that it is fine also as we only use that once).

Reply:
Ah good to know that. I think in this case is fine since its something used just here, but it would be useful in other cases, like specializing it for the Scene or the Model to easily log their status.